docs/migration_ios.md

```diff
@@ -99,6 +99,48 @@ This method is always called when the SDK is initiated successfully, regardless
 This method will be called if the user migration has failed with an error.
 We highly recommend using this method to log the error in your logging/analytics tools of choice.
 
+## Example implementation of Delegate Protocol in the ExampleViewController:
+
+import KinDevPlatform
+
+class ExampleViewController: UIViewController, KinMigrationDelegate {
```

Comment:
Can we get the whitespace and line breaks consistent please.

Reply:
done